web-app/js/portal/filter/Filter.js

```diff
@@ -15,6 +15,8 @@ Portal.filter.Filter = Ext.extend(Object, {
         this.displayLabel = filterDetail.label;
         this.type = filterDetail.type;
         this.visualised = filterDetail.visualised;
+        this.wmsStartDateName = filterDetail.wmsStartDateName;
+        this.wmsEndDateName = filterDetail.wmsEndDateName;
```

Comment:
Change made @anguss00. See below.

I didn't change any of the current usages (which currently use the property instead of the newly-added getter method) because they will basically all change in the refactoring.